### DISCUSSION TITLE
2022/08/kode-perintah-dasar-termux

### DISCUSSION THREAD

**Discussion by giscus** [madi10/MANTANKODE#28] [General]
# 2022/08/kode-perintah-dasar-termux

Materi singkat tentang perintah dasar yang dapat di gunakan pada apk Termux di Android

https://www.mantankode.net/2022/08/kode-perintah-dasar-termux.html

**Comment by tdtct** [comment]
Bang ajarin donk puh sepuh

**Reply by madi10** [reply]
ajarin apa bang...
